Makefile

```
@@ -3,9 +3,9 @@
 # Set the following variables before building:
 # Path to original CalculiX source (e.g. $(HOME)/ccx_2.xx/src )
 CCX_VERSION			= 2.20
-CCX             = $(HOME)/CalculiX/ccx_$(CCX_VERSION)/src
+CCX             = CalculiX/ccx_$(CCX_VERSION)/src
```

Why is `$(HOME)` removed? Wouldn't this break the path to find the CalculiX sources?

---

Good catch. Sorry this was a local change of mine.

```suggestion
CCX             = $(HOME)/CalculiX/ccx_$(CCX_VERSION)/src
```